### ISSUE TITLE
How to get out last softmax's input

### ISSUE THREAD

**Issue by andrisecker** [keras-team/keras#9701]
Is there an easy way to get out the **input** of the last softmax activation function?
I have bigger (512) Dense layer before my classification layer (Dense layer with 12 neurons, with softmax activation) and want to get out the input of the softmax function. 

```python
model = Sequential()
# ...
# Layer 5
model.add(Dense(512, activation="relu",
                  kernel_initializer="he_uniform", bias_initializer="zeros",
                  kernel_regularizer=l2(l**2*(0.7) / (2.*N))))
model.add(Dropout(0.3))
# Layer 6
model.add(Dense(12, activation="softmax",
                  kernel_initializer="he_uniform", bias_initializer="zeros"))
``` 

By doing `K.function([model.layers[0].input, K.learning_phase()], [model.layers[-1].input])`  I got an (1, 512) array (just as `model.layers[-2].output`) - but I want an (1, 12) array, just as `model.layers[-1].output`

PS: I've tried to add a `lambda x: x` debug layer, but that has the same (512) shape as the layer before

**Comment by tRosenflanz** [comment]
Should do it. In future Stack Overflow is better for implementation questions: 
```
model = Sequential()
# ...
# Layer 5
model.add(Dense(512, activation="relu",
                  kernel_initializer="he_uniform", bias_initializer="zeros",
                  kernel_regularizer=l2(l**2*(0.7) / (2.*N))))
model.add(Dropout(0.3))
# Layer 6
model.add(Dense(12,
                  kernel_initializer="he_uniform", bias_initializer="zeros"))
model.add(Activation('sigmoid')) 
```
